packages/velo-external-db-core/lib/schema_hooks_utils.js

```diff
@@ -0,0 +1,68 @@
+
+const hooksForAction = (action) => {
+    const hooksForAction = {
+        beforeList: ['beforeAll', 'beforeRead', 'beforeList'],
+        afterList: ['afterAll', 'afterRead', 'afterList'],
+        beforeListHeaders: ['beforeAll', 'beforeRead', 'beforeListHeaders'],
+        afterListHeaders: ['afterAll', 'afterRead', 'afterListHeaders'],
+        beforeFind: ['beforeAll', 'beforeRead', 'beforeFind'],
+        afterFind: ['afterAll', 'afterRead', 'afterFind'],
+        beforeCreate: ['beforeAll', 'beforeWrite', 'beforeCreate'],
+        afterCreate: ['afterAll', 'afterWrite', 'afterCreate'],
+        beforeColumnAdd: ['beforeAll', 'beforeWrite', 'beforeColumnAdd'],
+        afterColumnAdd: ['afterAll', 'afterWrite', 'afterColumnAdd'],
+        beforeColumnRemove: ['beforeAll', 'beforeWrite', 'beforeColumnRemove'],
+        afterColumnRemove: ['afterAll', 'afterWrite', 'afterColumnRemove']
+    }
+    return hooksForAction[action]
+}
+
+const Operations = {
+    LIST: 'list',
+    LIST_HEADERS: 'listHeaders',
+    FIND: 'find',
+    CREATE: 'create',
+    COLUMN_ADD: 'columnAdd',
+    COLUMN_REMOVE: 'columnRemove',
+}
+
+const Actions = {
+    BeforeList: 'beforeList',
```

Comment:
I think you should think of a convention. Here the keys are in UpperCamelCase and in `Operations` they are in CONST_CASE. I would align